packages/react-components/react-virtualizer/src/components/Virtualizer/renderVirtualizer.tsx

```diff
@@ -20,3 +21,11 @@ export const renderVirtualizer_unstable = (state: VirtualizerState) => {
     </React.Fragment>
   );
 };
+
+export const renderVirtualizerChildPlaceholder = (child: ReactNode, index: number) => {
```

Comment:
This isn't explicitly a requirement for dynamic items, but this does give us major performance optimizations during heavy element scroll as React.Suspense does a much better job at managing the DOM load over default react DOM changes.

Reply:
Only suspense enabled data loading will actually activate the `Suspense` component, I'm not sure that we should be optimizing here. Could the user simply render the `Suspense` component themselves?

Reply:
I mostly added this as it has a non-negligible effect on scroll performance during fast scrolls with both basic and heavier elements (even without data loading, perhaps some kind of interaction there with queued intersection observer calls), maybe this can just be a recommendation for heavier elements but it seemed like a net-positive change.